src/main/resources/org/jenkinsci/test/acceptance/docker/fixtures/JavaContainer/Dockerfile

```diff
@@ -5,11 +5,12 @@
 # cf. SshdContainer/Dockerfile
 FROM jenkins/sshd:ff2527313f5c
 
+ENV JDK_VERSION 8u144-b01-2
 RUN apt-get update && \
     apt-get install --no-install-recommends -y \
         software-properties-common=0.96.24.17 \
-        openjdk-8-jdk=8u144-b01-2 \
+        openjdk-8-jre-headless=$JDK_VERSION \
+        openjdk-8-jdk-headless=$JDK_VERSION \
```

Comment:
To be really safe I would need to specify every transitive dependency, but there are dozens of them and this seems impractical for now.